app/views/finders/_search_results.html.erb

```diff
@@ -0,0 +1,80 @@
+<ul class="finder-results" data-module="track-click">
+  <% if local_assigns[:display_grouped_results] %>
```

Comment:
Might want to get someone from the team responsible for the grouping stuff to review this.

Reply:
cc @steventux